### ISSUE TITLE
Help with CReLU in Pytorch

### ISSUE THREAD

**Issue by sinAshish** [Andrew-Qibin/SeeNet#4]
Hi @Andrew-Qibin , I am try to implement your paper in pytorch. Since I am a bit new to pytorch, Can you please help with the CRelU layer in pytorch and the label for mask_r2 layer?

**Comment by houqb** [comment]
You can use torch.where() or get binary tensor directly using B_A = A > theta

**Comment by tjufan** [comment]
Hi @Andrew-Qibin, the released code appears to be the code for the inference stage, not the training stage, can you release the code of training stage? Otherwise, i am also confused with the CReLU layer. is the label a number like 1, 2, 3, not a 2-D map? , and why do you deal with the mask like this when label[i] == 1?
        mask = bottom[1].data[0].copy()
        mask[mask < 0] = 0
        for i in range(20):
            if int(label[i]) == 1:
                ma = np.max(mask[i])
                mi = np.min(mask[i])
                mask[i] = (mask[i] - mi) / (ma - mi + 1e-8)
            else:  # 0
                mask[i,...] = 0
